rasterio/plot.py

```diff
@@ -10,6 +10,7 @@
 import logging
 import warnings
 
+import numpy as np
 import rasterio
```

Comment:
@ColinTalbert it's not documented anywhere yet, but we prefer 3 import sections: standard libs, non-standard libs, and the rasterio namespace. One empty line between each, each section sorted as you've done.